tests/rt.conf

```diff
@@ -21,10 +21,18 @@
 
 ### Intel Tests ###
 ### S2S tests ###
-COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 |          | fv3 |
+COMPILE | s2swa_32bit  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_v17_coupled_p8_ugwpv1 |          | fv3 |
 RUN | cpld_control_p8_mixedmode                         |    - noaacloud                     | baseline |
-RUN | cpld_control_gfsv17                               |                                    | baseline |
-RUN | cpld_control_gfsv17_iau                           |                                    | baseline | cpld_control_gfsv17
+
+#GFS
+COMPILE | s2swa_32bit_pdlib  | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON | + hera orion cheyenne wcoss2 acorn | fv3 |
+RUN | cpld_control_gfsv17                               | + hera orion cheyenne wcoss2 acorn | baseline |
+RUN | cpld_control_gfsv17_iau                           | + hera orion cheyenne wcoss2 acorn | baseline | cpld_control_gfsv17
+RUN | cpld_restart_gfsv17                               | + hera orion cheyenne wcoss2 acorn |          | cpld_control_gfsv17 
+RUN | cpld_mpi_gfsv17                                   | + hera orion cheyenne wcoss2 acorn |          | cpld_control_gfsv17
```

Comment:
Please remove cpld_control_gfsv17 in line 32.

Reply:
I don't understand why we'd remove that as the baseline for cpld_mpi_gfsv17 is cpld_control_gfsv17 ?

Reply:
OK, please read the code. The baseline is specified in the test. Please see:
https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/tests/cpld_mpi_p8#L7
Please see the line: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/rt.conf#L36

Reply:
I had the definition of dependency to also mean baseline, but now I see that's not the case.  Thanks for helping me understand and I'll send Mike a PR to fix this now.

Reply:
@mdtoyNOAA PR for this is here: https://github.com/mdtoyNOAA/ufs-weather-model/pull/6